src/forms/labels/labeled-field.js

```diff
@@ -88,23 +91,25 @@ function LabeledField({
   children,
   hideErrorLabel,
   label,
+  as: Wrapper,
   ...rest
 }) {
   const { name } = input
   const { touched, invalid } = meta
   return (
-    <fieldset
-      className={classnames(className, {
+    <Wrapper
+      className={classnames(className, 'field-wrapper', {
         error: hasInputError({ touched, invalid }),
         disabled: rest.disabled,
       })}
+      data-testid="field-wrapper"
```

Comment:
Should this be dynamic based on the input's name? If you have a test with multiple inputs, it might be challenging to locate a particular container. Not sure there's a use case for that though 🤷

Reply:
decided to ditch `data-testid` and instead use `{ container }` from `render` results as you suggested! https://testing-library.com/docs/react-testing-library/api/#container-1